### ISSUE TITLE
Pytorch-Lightning 中的 total steps的变化

### ISSUE THREAD

**Issue by rentainhe** [rentainhe/mini-classification#5]
https://github.com/PyTorchLightning/pytorch-lightning/discussions/9179#discussioncomment-1345219
> The global_step indicates how many times the parameters are updated. Therefore, with gradient accumulation and automatic optimization enabled, the global step is only incremented after the gradients are accumulated and not after every training_step is run

Pytorch-Lightning中global steps的变化是根据你参数的更新来决定的，当gradient accumulation和自动优化打开的时候，global step是在每次梯度累积更新后再update的，而不是根据你的training_step
